[Excel] (Custom Functions) Adjust custom functions overview article (#2390)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-troubleshooting.md

---

> ms.date: 12/29/2020

Reviewer: ```suggestion
ms.date: 01/08/2020
```

For more information on this process, see [Associating function names with JSON metadata](../excel/custom-functions-json.md#associating-function-names-with-json-metadata).

> ## Known issues

Reviewer: There's something slightly off about having  "Common issues" precede "Known issues". The latter is using "issues" to mean bugs or deficiencies, whereas the former is referring to common design pitfalls.

Author: Do you have alternate phrasing suggestions for either location? "Known issues" is oblique but it's how we title the bugs sections in add-ins docs and is in line with the industry standard, so I'm inclined to change "Common issues" instead. What do you think of "Common problems"? It avoids the confusing usage of "issues" but unfortunately is similarly vague.

Reviewer: Agreed that "Common issues" is the part to change. I like "Common problems", though perhaps "Common problems and solutions" sounds more enticing.

Author: Great idea!

> See a list of additional known issues by checking the [Excel Custom Functions GitHub repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).

Reviewer: I'm not sure these are "additional" issues, rather than a complete list.
```suggestion
Known issues are tracked and reported in the [Excel Custom Functions GitHub repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).
```
